en/docs/learn/api-controller/getting-started-with-wso2-api-controller.md

```diff
@@ -813,4 +813,22 @@ Run the following CTL command to change the default location of the export direc
         - Required :   
             `--export-directory`: Path to directory where APIs should be saved.   
             Default : `/home/.wso2apictl/exported`
+            
+            
+## Import SSL Certificate for Secure HTTP Communication with API Manager
+
+Different environments of API Manager can have different SSL certificates for secure HTTP communications. The default
+certificate of WSO2 API Manager is a self-signed certificate and in production environments, it is advised to use a
+different certificate than the default.   
+
+If the certificate is the default WSO2 certificate or a CA-signed certificate of a CA (Certificate Authority) trusted by
+the OS, these certificates will be imported by default to the controller. If the CA or the certificate is new or does
+not get imported by default, you can add the certificate to the ```/home/.wso2apictl/certs``` directory. The
```

Comment:
Now 3.2 onwards, we are allowing them to customize the root location of the .wso2apictl directory, so shall we reflect that here too?

Reply:
The users can locate the .wso2apictl directory to another folder by setting an env variable, and of course the default will be the HOME directory